docs/guide/integrations/nuxt3.md

```diff
@@ -0,0 +1,314 @@
+
+### Create Nuxt 3 application
+
+Run the following command to create a Nuxt 3 application:
+
+```sh
+npx nuxi init nuxt3-app-vue-i18n
+```
+
+We will have run the above commmand, it will be created  Nuxt 3 initial project with the following directory structure:
+
+```
+cd nuxt3-app-vue-i18n
+tree -L 1
+.
+├── README.md
+├── app.vue
+├── node_modules
+├── nuxt.config.ts
+├── package-lock.json
+├── package.json
+└── tsconfig.json
+```
+
+### Install Vue I18n
+
+Install Vue I18n with the following command:
+
+```sh
+npm install --save-dev vue-i18n
+```
+
+### Setup Nuxt plugin
+
+We will set up the Nuxt plugin.
+
+Create a `plugins` directory as follows:
+
+```sh
+mkdir plugins
+```
+
+Next, create a Nuxt plugin file to implement for setting up Vue I18n.
+
+```sh
+touch plugins/i18n.ts
+```
+
+When we have created it, edit it with the below codes:
+
+```ts
+import { createI18n } from 'vue-i18n'
+
+export default defineNuxtPlugin(({ vueApp }) => {
+  const i18n = createI18n({
+    legacy: false,
+    globalInjection: true,
+    locale: 'en',
+    messages: {
+      en: {
+        hello: 'Hello, {name}!'
+      }
+    }
+  })
+
+  vueApp.use(i18n)
+})
+```
+
+We aim to set up Vue I18n in the implementation here.
+
+Configure up locale resources to localize a Nuxt 3 application is described in the next section.
+
+### Run the Nuxt 3 application
+
+Let's see if Vue I18n works with Nuxt 3.
+
+We will edit `app.vue` of the setup Nuxt 3 application as follows:
+
+```diff
+ <template>
+   <div>
+-   <NuxtWelcome />
++   <h1>{{ $t('hello', { name: 'vue-i18n' }) }}</h1>
+   </div>
+ </template>
+```
+
+We have edited and saved, run the following command to run the Nuxt 3 application at local:
+
+```sh
+npm run dev
+```
+
+After we will run the command and access to `http://localhost:3000`, you can see the display similar to the following:
+
+![Nuxt3 setup](/nuxt3-setup.png)
+
+## Localize your Nuxt 3 application
+
+So far we have been able to integrate Vue I18n into our Nuxt 3 application.
+Next, we will implement language switching and import locale resources from outside.
+
+By implementing language switching, we can i18n our Nuxt 3 application. Also, when we will be separating locale resoruces from the source code and externalizing them, we can be localized in a separate workflow using the Localization service.
+
+In the following sections, we will work on Nuxt 3 applications that support English, French, and Japanese.
+
+### Add language switching
+
+We will add language switching feature to `app.vue` as follows:
+
+```diff
+ <template>
+   <div>
+     <h1>{{ $t('hello', { name: 'vue-i18n' }) }}</h1>
++    <form>
++      <label for="locale-select">{{ $t('language') }}: </label>
++      <select id="locale-select" v-model="$i18n.locale">
++        <option value="en">en</option>
++        <option value="fr">fr</option>
++        <option value="ja">ja</option>
++      </select>
++    </form>
++  </div>
```

Comment:
`</div>` is already in `app.vue`, not a new addition, so no plus sign is needed at the beginning of the line.